dms_field/models/dms_access_group.py

```diff
@@ -22,6 +28,15 @@ def _selection_reference_value(self):
         )
         return [(model.model, model.name) for model in models]
 
+    @api.depends("dms_field_ref")
+    def _compute_company_id(self):
+        for item in self.filtered("dms_field_ref"):
```

Comment:
For the rest of the records:

```suggestion
        self.company_id = False
        for item in self.filtered("dms_field_ref"):
```